src/mixer/basetrackplayer.cpp

```diff
@@ -218,7 +218,7 @@ void BaseTrackPlayerImpl::loadTrack(TrackPointer pTrack) {
         QListIterator<CuePointer> it(trackCues);
         while (it.hasNext()) {
             CuePointer pCue(it.next());
-            if (pCue->getType() == mixxx::CueType::Loop) {
+            if (pCue->getType() == mixxx::CueType::Loop && pCue->getHotCue() == Cue::kNoHotCue) {
```

Comment:
If we're going to import Serato loops with hotcue numbers, then this would make Mixxx completely ignore all of them. Don't we want Mixxx to use one of them?

Reply:
We could also load loops with a hotcue number if no loop without one exists. That would require more changes though.

Reply:
Is the order of iteration over the CuePointers guaranteed to match the hotcue number order? If so, we can just break after the first CuePointer. If not, yeah I think more changes are needed.

Reply:
I don't know. I guess the order is somewhat random. I need to check, the DB query probably happens in CueDAO.

Reply:
We probably shouldn't depend on that here anyway, even if happens to work now.

Reply:
Done. https://github.com/mixxxdj/mixxx/pull/2678/commits/7541fe5f2cb07c6a39cc0e188938dddb36c70715